### PR TITLE
Added manage bigquery and small spacing in existing bigquery setup

### PR DESCRIPTION
**Pull request by tommyjepsen**

### CODE REVIEW

File: managedbigquery.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# What is BigQuery by Weld
+
+We are offering a managed BigQuery by Weld free up til 1TB Stored Data for you to easily get started.
```
Comment (stefanbugge) [inline managedbigquery.md:3]:
```suggestion
We are offering a managed BigQuery by Weld free up to 1TB stored data for you to easily get started.
```

**Review by stefanbugge — COMMENTED** [review]
lige et par smårettelser

File: managedbigquery.md (hunk 1 above)
Comment (sarawungsung) [reply managedbigquery.md]: We are offering a managed BigQuery by Weld for free up to 1TB stored data for you to easily get started.

File: managedbigquery.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# What is BigQuery by Weld
```
Comment (stefanbugge) [inline managedbigquery.md:1]: Kan vi rename filen til `weld-bigquery.md` så følger det samme navne-konvention som i frontend implementationen.?

**Pull request merged**